admin/class-admin.php

```diff
@@ -146,13 +146,7 @@ public function options_validate( $input ) {
 
 		$input['mincomlength']  = (int) $input['mincomlength'];
 		$input['redirect_page'] = (int) $input['redirect_page'];
-
-		if ( isset( $input['clean_emails'] ) ) {
-			$input['clean_emails'] = true;
-		}
-		else {
-			$input['clean_emails'] = true;
-		}
+  		$input['clean_emails']  = true;
```

Comment:
Hmm this basically means you can't disable clean_emails, so it's more a bug than that this solves anything.